.travis.yml

```diff
@@ -48,7 +48,7 @@ jobs:
     - script: bin/builder_run bin/check_golint
       name: "golint check"
 
-    - script: bin/check_goimports
+    - script: bin/builder_run bin/check_goimports
```

Comment:
I'd try to avoid using extra pre-build images for simple task. 
Maybe it's better to do `go get goimports` instead of running it in extra docker container?

Reply:
And looks like direct call works a bit faster on my system:
![image](https://user-images.githubusercontent.com/8612618/47021786-f0733780-d17d-11e8-9b78-c581a467d487.png)

Reply:
This ^ was not counting time for image downloading.

Reply:
Your points are absolutely valid, but:
1. It's not improvement or feature, its a hotfix - intention was to fix it asap with minimal effort as build functionality is currently broken
2. golint is also go getted inside builder and is run the same way, maybe we can get rid of this builder completely when migrating to go style CI